doc/source/amqp1d.rst

```diff
@@ -0,0 +1,37 @@
+===================
+Amqp1d Daemon Usage
+===================
+
+What Is It?
+===========
+`AMQP 1.0`_ Advanced Messaging Queuing Protocol 1.0 is ISO/IEC 19464.
+
+Gnocchi provides a daemon `gnocchi-amqp1d` that is compatible with the `AMQP
+1.0`_ (Advanced Messaging Queuing Protocol 1.0 (ISO/IEC 19464)) protocol and
+can listen to |metrics| sent over the network via amqp1 `collectd`_  plugin ,
+named `amqp1`_.
+
+.. _`amqp1`: https://github.com/collectd/collectd/blob/master/src/amqp1.c
+.. _`collectd`: https://github.com/collectd/collectd
+.. _`AMQP 1.0`: https://www.amqp.org/resources/specifications
+
+`amqp1` collectd write plugin enables collectd output to be sent to an Advanced
+Messaging Queuing Protocol 1.0 intermediary such as the Apache Qpid Dispatch
+Router or Apache Artemis Broker.
+
+
+How It Works?
+=============
+In order to enable amqp1d support in Gnocchi, you need to configure the
```

Comment:
missing installation instruction with the flavor

Reply:
done